Core/src/main/java/com/paypal/android/core/analytics/AnalyticsClient.kt

```diff
@@ -0,0 +1,30 @@
+package com.paypal.android.core.analytics
+
+import android.util.Log
+import com.paypal.android.core.Http
+import com.paypal.android.core.HttpRequestFactory
+import java.util.UUID
+
+internal class AnalyticsClient(
```

Comment:
Nitpick / naming comment - I think we use the word `Client` a lot, and it's meaning is a bit conflated. I opted for `AnalyticsService` on iOS, since the android feature modules aren't interfacing directly with this `Analytics` layer & it is what we use on BT.

I don't feel too strongly, but it would be nice if we can align on the same name and decide what makes something a `Client`.

Reply:
Agree, perhaps we could reserve `Client` to the classes the the merchant should interact with.